recipe/meta.yaml

```diff
@@ -42,7 +42,7 @@ requirements:
         - mingwpy  # [win and not py3k]
 {% if osx_variant == "pocl" %}
         - osx-pocl-opencl
-        - pocl >=1.1
+        - pocl >=1.1_1
```

Comment:
It seems specifying the build number in inequalities is not supported

Reply:
`!=1.1 0` works, but `>=1.1,!=1.1 0` doesn't.

Reply:
OK. Thanks for following up. I'll just keep it simple, with `>=1.1`, and rely on the broken tag to keep `1.1_0` out.